app/main.py

```
@@ -3071,6 +3324,299 @@ async def batch_optimize_images(
         raise HTTPException(status_code=500, detail=str(e))
 
 
+        organization_id = org.organizations[0].id
```

Ruff (0.14.14)

[error] Local variable `organization_id` is assigned to but never used

Remove assignment to unused variable `organization_id`

(F841)